site/js/worklets/theme.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Site Theme setter and getter
+ */
+export class Theme {
```

Comment:
`worklets` have a very specific meaning; it's a [light-weight Web Worker used as part of the rendering pipeline](https://developer.mozilla.org/en-US/docs/Web/API/Worklet). This appears to be a component like our other components. Please move and rename as appropriate

Reply:
moved to `components`